src/main/java/com/paneedah/weaponlib/network/packets/BloodPacketClient.java

```diff
@@ -11,14 +13,16 @@
 
 public class BloodPacketClient implements CompatibleMessage {
 
+	public static float rotationYaw;
+
 	double x, y, z, velx, vely, velz;
 
 	public BloodPacketClient() {}
 	
 	public BloodPacketClient(double x, double y, double z, double velX, double velY, double velZ) {
-		this.x = x;
+		this.x = x - CompatibleMathHelper.cos(rotationYaw / 180.0F * (float) Math.PI) * (EntityProjectile.isAim ? 0f : 0.16F);
 		this.y = y;
-		this.z = z;
+		this.z = z - CompatibleMathHelper.sin(rotationYaw / 180.0F * (float) Math.PI) * (EntityProjectile.isAim ? 0f : 0.16F);
```

Comment:
I think there is a little logic error here, with the ternary operator the first option is the true one, so this should be (isAim ? 0.16F : 0)